subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/screenshot/ScreenshotCapturerImpl.kt

```diff
@@ -14,7 +16,14 @@ import java.io.FileOutputStream
 /**
  * Source: https://github.com/square/spoon/blob/master/spoon-client/src/main/java/com/squareup/spoon/Screenshot.java
  */
-class ScreenshotCapturerImpl(private val outputDirectory: Lazy<File>) : ScreenshotCapturer {
+
+class ScreenshotCapturerImpl(private val reportFileProvider: ReportFileProvider) : ScreenshotCapturer {
```

Comment:
screenshot capturer doesn't want to know about report file provider it wants to know about `dir` let's give it to it